contracts/ERC20Interface.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.0;
+pragma solidity ^0.4.21;
 
 contract ERC20Interface {
```

Comment:
Essentially symbols, decimals and name are not a part of ERC20 standard. Also we do not appreciate commented code in repositories.

Reply:
Just got Interface contract from original Remme token contract, and implement them